hw2/04-background.html

```diff
@@ -1,29 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>04 - Background</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="04-background.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
-    </main>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>04 - Background</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="04-background.js"></script>
+</head>
+
+<body class="vh-100 bg-primary h-100 pt-4">
+  <main class="card mx-auto w-50 p-3">
+    <h1>Exercise 04 - Background</h1>
+    <p>
+      The background color will change every few seconds (as indicated below).
+      You can stop this behavior and you can also change how often the
+      background color changes.
+    </p>
+    <div>
+      <label for="interval">Background change interval (seconds): </label>
+      <input type="number" id="interval" />
+    </div>
+
+    <div class="mt-4">
+      <button class="btn btn-primary" id="cta">Start</button>
```

Comment:
An id of "cta" is not very descriptive name.

Reply:
hmmm, isn't `cta` (Call to action) a well known term among the web dev community ❓ Like how the `btn` is known as button

Reply:
That's true in sense.

Reply:
That's a good point. I think it may be a personal preference of mine to be quite verbose when naming. It leaves less room for misinterpretation.

Reply:
no that makes sense, I should have done it too, but I guess old habit dies hard sometimes...